pnpm-lock.yaml

```diff
@@ -118,25 +118,25 @@ dependencies:
     version: 9.7.2(react-dom@18.2.0)(react@18.2.0)
   '@react-stately/table':
     specifier: ^3.9.1
-    version: 3.11.0(react@18.2.0)
+    version: 3.9.1(react@18.2.0)
```

Comment:
can we revert this? Why are all these libraries getting downgraded in a pr that doesn't touch the lockfile?